input/intro-notes/StructureDefinition-au-core-patient-intro.md

```diff
@@ -48,4 +48,7 @@ By making Individual Gender Identity a Must Support data element, this profile e
   - Where the workflow does not support obtaining a gender identity value, it may be represented by sending the code "not-asked"
 - When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/) and [AU Base Patient](https://build.fhir.org/ig/hl7au/au-fhir-base//StructureDefinition-au-patient.html)
 
+<p class="request-for-feedback">Following the TDG agreement to set Must Support on parts of HumanName in AU Core Patient (noting the existing Must Support on Patient.name.use), specific feedback is sought on other parts of HumanName that should be supported.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44818">FHIR-44818</a>.</p>
```

Comment:
Replace
Following the TDG agreement to set Must Support on parts of HumanName in AU Core Patient (noting the existing Must Support on Patient.name.use), specific feedback is sought on other parts of HumanName that should be supported.

To be 
Specific feedback is sought on the proposal to add Must Support to parts of Patient.name in addition to Patient.name.use.